polus-vector-label-plugin/src/main.py

```diff
@@ -0,0 +1,101 @@
+from bfio import  BioWriter , OmeXml,JARS,LOG4J
+#from bfio import  BioWriter
+import bioformats
+import javabridge as jutil
+import argparse, logging, sys
+import numpy as np
+from pathlib import Path
+import re
+import zarr
+import mask
+
+if __name__=="__main__":
+    # Initialize the logger
+    logging.basicConfig(format='%(asctime)s - %(name)-8s - %(levelname)-8s - %(message)s',
+                        datefmt='%d-%b-%y %H:%M:%S')
+    logger = logging.getLogger("main")
+    logger.setLevel(logging.INFO)
+
+    ''' Argument parsing '''
+    logger.info("Parsing arguments...")
+    parser = argparse.ArgumentParser(prog='main', description='Cellpose parameters')
+    
+    # Input arguments
+
+    parser.add_argument('--inpDir', dest='inpDir', type=str,
+                        help='Input image collection to be processed by this plugin', required=True)
+    parser.add_argument('--flow_threshold', required=False,
+                        default=0.4, type=float, help='flow error threshold, 0 turns off this optional QC step')
+    parser.add_argument('--cellprob_threshold', required=False,
+                        default=0.0, type=float, help='cell probability threshold, centered at 0.0')
+    # Output arguments
+    parser.add_argument('--outDir', dest='outDir', type=str,
+                        help='Output collection', required=True)
+    
+    # Parse the arguments
+    args = parser.parse_args()
+
+    inpDir = args.inpDir
+
+
+    logger.info('inpDir = {}'.format(inpDir))
+
+    outDir = args.outDir
+    logger.info('outDir = {}'.format(outDir))
+
+    cellprob_threshold = args.cellprob_threshold
+    flow_threshold= args.flow_threshold
+
+    # Surround with try/finally for proper error catching
+    try:
+        # Start the javabridge with proper java logging
+        logger.info('Initializing ...')
+        log_config = Path(__file__).parent.joinpath("log4j.properties")
+   #     jutil.start_vm(args=["-Dlog4j.configuration=file:{}".format(str(log_config.absolute()))],class_path=JARS)
+        jutil.start_vm(args=["-Dlog4j.configuration=file:{}".format(LOG4J)],class_path=JARS)
```

Comment:
This can be removed since the javabridge is not installed in the container you are using.